### PR TITLE
[MAT-1042][MAT-1535] Parsing Population Set and display values for Measure Dashboard

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
- Population Set parsing logic to support front-end calculation and display.

- Parsing logic for display attributes: CMS ID and measure title.
**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this [MAT-1042](https://jira.cms.gov/browse/MAT-1042), [MAT-1535](https://jira.cms.gov/browse/MAT-1535)
- [ ] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/measure-loader/bundle_loader.rb [hunk 1]
```diff
@@ -89,22 +89,28 @@ def create_measure(measure_bundle)
...
 
       cqm_measure.cql_libraries = parse_cql_elm(libraries, fhir_measure.name.value, fhir_measure.version.value)
       elms = cqm_measure.cql_libraries.map(&:elm)
       elm_value_sets = ValueSetHelpers.unique_list_of_valuesets_referenced_by_elms(elms)
       cqm_measure.value_sets = ValueSetHelpers.make_fake_valuesets_from_drc(elms, @vs_model_cache)
       cqm_measure.value_sets.concat(@value_set_loader.retrieve_and_modelize_value_sets_from_vsac(elm_value_sets)) if @value_set_loader.present?
-      fhir_value_sets = cqm_measure.value_sets.map(&:fhir_value_set).compact
-      cqm_measure.source_data_criteria = libraries.map{|lib| lib.create_data_elements(fhir_value_sets)}.flatten
+
+      # TODO: uncomment once we have TS models integrated in bonnie.
```
Comment (serhii-ilin) [inline lib/measure-loader/bundle_loader.rb:110]: Why is it blocked with TS models?

Comment (adongare) [reply lib/measure-loader/bundle_loader.rb:110]: when we try to convert frontend models into thorax model this complains because frontend measure is still QDM measure model(currently it) and has different structure for source_data_criteria

Comment (jkotanchik-SB) [reply lib/measure-loader/bundle_loader.rb:110]: The `cqm-models` JS is unable to process the new source data criteria model and the resulting exceptions were preventing the fhir measure from displaying on the dashboard.

Comment (serhii-ilin) [reply lib/measure-loader/bundle_loader.rb:110]: Ah.. I didn't realize bonnie-fhir is not switched to typescript models
